build/yamls/flow-visibility.yml

```diff
@@ -5009,6 +4916,26 @@ spec:
             name: clickhouse-configmap-volume
           - mountPath: /var/lib/clickhouse
             name: clickhouse-storage-volume
+        - env:
+          - name: CLICKHOUSE_USERNAME
+            valueFrom:
+              secretKeyRef:
+                key: username
+                name: clickhouse-secret
+          - name: CLICKHOUSE_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                key: password
+                name: clickhouse-secret
+          - name: DB_URL
+            value: tcp://clickhouse-clickhouse.flow-visibility.svc:9000
```

Comment:
there is no need to use the svc DNS anymore. They are in the same network namespace now, so you can just use `tcp://localhost:9000`

another question related to @dreamtalen's comment above: I don't see a way to change the port to something other than 9000 for the ClickHouse server in this manifest. How would a user be able to do that?

Reply:
Updated the DNS.

Reply:
Previously we do not provide a clear way to change the port. I updated the code to allow that. But currently the user might need multiple steps to complete the change.

I'm not sure whether there are use cases that the ports only exposed to the ClickHouse pod required to be changed, but if the user would like to change the port exposed in the pod, they need to update 

- The corresponding port in the `hostTemplates` in `clickhouse.yml` 
- The `DB_URL` env used by the monitor in `clickhouse.yml` 
- The port used to connect to the database when initializing the ClickHouse server in `create_table.sh`

If the user would like to change the port exposed out of the pod, which is the `clickhouse-clickhouse` service, they need to update 

- The corresponding port in the `serviceTemplates` in `clickhouse.yml`
- The port used by the Grafana defined in `datasouce_provider.yml`
- The `databaseURL` used by the Flow Aggregator defined in flow-aggregator.conf, which is introduced in PR #3196 

I think it is not convenient for users to do all these changes just to use another port other than 9000. Would like to hear from @heanlan about suggestions on what we can do to make the change easier.